rusk/src/lib/chain/rusk.rs

```diff
@@ -210,6 +236,17 @@ impl Rusk {
             }
         }
 
+        let r = Migration::migrate(
+            self.migration_height,
+            session,
+            block_height,
+            provisioners,
+        );
+        if r.is_err() {
+            info!("MIGRATION RESULT={:?}", r);
+        }
+        let session = r.expect("migration should succeed");
+
```

Comment:
This needs to be removed, since the migration is already done in the accept at line 220